### ISSUE TITLE
YAML indentation is 4 spaces

### ISSUE THREAD

**Issue by nickjer** [spf13/spf13-vim#789]
By default all my *.yml files are being indented by 4 spaces using `spf13-vim`. I'd like it to be 2 spaces.

I observed in the `.vimrc` file it is explicitly setting indentation to be 2 spaces for *.yml files:

https://github.com/spf13/spf13-vim/blob/3.0/.vimrc#L259

But I am seeing 4 space indentation. Is there a specific plugin that is changing it to 4? And how can I disable that plugin's feature.

**Comment by jasonshell** [comment]
The below snippet is in my .vimrc and properly indents .yml files in macvim.

``` vim
" yaml indentation
au FileType yaml setlocal tabstop=2 expandtab shiftwidth=2 softtabstop=2
```

It appears your yaml rule has omitted `tabstop=2` which vim uses for yaml to determine how many spaces per indent.

**Comment by ghost** [comment]
seems like this was solved?

**Comment by BretFisher** [comment]
Could it be that the actual FileType should be `yaml` but the spf13-vim/.vimrc has it as `yml` on several lines? If I manually set FileType to `yml` it won't parse correctly, so seems it should always be `yaml`.

**Comment by zx1986** [comment]
@BretFisher yeah, I got this issue too. `*.yml` always start a new line with `tab` indent ...
